util/wrappergen/wrappergen.py

```diff
@@ -52,6 +52,16 @@ def gen_file(cfg, tpl, target_path: str, file_name: str) -> None:
     return
 
 
+def gen_chisel_file(chisel_path, chisel_param, gen_path):
+
+    # Call chisel environment and generate the system verilog file
+    cmd = f' cd {chisel_path} && sbt \
+        "runMain {chisel_param} {gen_path}"'
+    os.system(cmd)
+
+    return
+
+
```

Comment:
nit: At some point we should merge this with chiselgen

Reply:
I obliterated that tiny piece 😆